target/fail2ban/jail.local

```diff
@@ -8,7 +8,7 @@ bantime = 1w
 findtime = 1w
 
 # "maxretry" is the number of failures before a host get banned.
-maxretry = 2
+maxretry = 10
```

Comment:
I know, @polarathene argued that it's unlikely to guess a reasonable long password, which I second.

IMO 10 retries are nevertheless too much:

- spambots will now try 10 times instead of 2 which will create significantly more failed login entrys in `mail.log` (makes it harder to read/analyze).
- I don't think a human will wrongly enter the password that many times.

I would keep it, or set it lower to 3 or 5.

Reply:
I actually kind of agree :) I will set it to 6, because I remember this was mentioned in the discussion as a recommendation.

Reply:
Done in [0211c59](https://github.com/docker-mailserver/docker-mailserver/pull/3288/commits/0211c59c47e2c10c20936fea0d7f09cc9afc9684)